markets/bfp-market/contracts/modules/SplitAccountConfigurationModule.sol

```diff
@@ -0,0 +1,61 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.11 <0.9.0;
+
+import {IERC165} from "@synthetixio/core-contracts/contracts/interfaces/IERC165.sol";
+import {OwnableStorage} from "@synthetixio/core-contracts/contracts/ownership/OwnableStorage.sol";
+import {ERC165Helper} from "@synthetixio/core-contracts/contracts/utils/ERC165Helper.sol";
+import {ERC2771Context} from "@synthetixio/core-contracts/contracts/utils/ERC2771Context.sol";
+import {ISplitAccountConfigurationModule} from "../interfaces/ISplitAccountConfigurationModule.sol";
+import {SplitAccountConfiguration} from "../storage/SplitAccountConfiguration.sol";
+import {ErrorUtil} from "../utils/ErrorUtil.sol";
+
+contract SplitAccountConfigurationModule is ISplitAccountConfigurationModule {
```

Comment:
You could probably put this as part of the PerpAccountModule since it's kinda where all the splitAccount related fn is located and imo this isn't enough functionality to warrant having its own module tbh.

Reply:
Yea sorta agree but it does follow the same pattern as whitelisting of hooks. The hooks stuff could be in the order module too